lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,66 @@
 import React from 'react';
+import PropTypes from "prop-types";
 
+import styled from "styled-components";
+
+const ActiveTab = styled.div` 
+  background-color: #fff;
+  color: #333;
+  border: 2px solid #333;
+  display: flex;
+  justify-content: none;
+  align-items: center;
+  flex-direction: row;
+  color: #fff;
+  background-color: #333;
+  margin: 0 5px;
+  padding: 2px 10px;
+  font-size: 12px;
+  letter-spacing: 2px;
+  cursor: pointer;
+  font-weight: bold;
+  &:hover{
+    text-decoration: underline;
+  }
+`
+const TaB = styled.div`
+  display: flex;
+  justify-content: none;
+  align-items: center;
+  flex-direction: row;
+  color: #fff;
+  background-color: #333;
+  margin: 0 5px;
+  padding: 2px 10px;
+  font-size: 12px;
+  letter-spacing: 2px;
+  cursor: pointer;
+  font-weight: bold;
+  &:hover{
+    text-decoration: underline;
+  }
+`
 const Tab = props => {
-  /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
-      if they match, the className should be: 'tab active-tab', 
-      if it is not it should just be 'tab'*/
+  /* Using your props, determine if this tab is selected, if it is, the className 
+    should be: 'tab active-tab', if it is not it should just be 'tab' */
+  const selectedClass = props.selectedTab;
   return (
-    <div
-      className={''}
-      onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
-      }}
-    >
-      {props.tab.toUpperCase()}
+    <div>
+      {(props.tab === selectedClass) ?
+        <ActiveTab onClick={(e) => { e.preventDefault; props.selectTabHandler(props.tab) }}>{props.tab.toUpperCase()}</ActiveTab>
```

Comment:
Instead of writing e.preventDefault here in the onClick, you could add it to the selectTabHandler function, changeSelected.